src/messagingSummaryItem/__snapshots__/index.unit.tsx.snap

```diff
@@ -42,11 +42,10 @@ exports[`Should render properly with read messages 1`] = `
     className="jsx-1542968643 kirk-item-rightAddon"
   >
     <div
-      className="jsx-1920606029 kirk-avatar--image kirk-avatar"
+      className="kirk-avatar--image sc-bwzfXH fylziy kirk-avatar"
```

Comment:
nit: Add the jest-styled-components library to avoid having the generated class names in the snapshots

Reply:
Done, I pushed it in a separate commit. I applied it in the Jest config so it should be global.